cycle/cycle.env

```diff
@@ -37,7 +37,7 @@ lsstsqre=7-stack-lsst_distrib-w_2024_25
 stack=w_2024_25
 # Base image for Rapid Analysis.
 # TODO: (DM-43475) Resync RA images with the rest of the summit.
```

Comment:
Is this TODO still needed?

Reply:
Yes, this is part of a different work and needs to be kept :+1:.